docs/overview/component/index-manager.md

```diff
@@ -0,0 +1,43 @@
+# Vald Index Manager
+
+Vald Index Manager is the component that controls the indexing process of Vald Agent Pods in the same Vald cluster.
+
+## Responsibility
+
+Vald Index Manager has a unique and simple role in preventing stop-the-world.
+
+It requires `Vald Discoverer` to fulfill its responsibility.
+
+## Features
+
+This chapter shows the main features to fulfill Vald Index Manager’s role.
+
+### Syncing data from Vald Discoverer
```

Comment:
I think it is not the main feature of index manager? 
I think it should be part of "control indexing process" section

Reply:
it is also correct.
why I wrote such as is `control indexing process` section will be too long if I include it.
And more, syncing from the discoverer is not implemented in the `controlling indexing process`, so I split.

Reply:
I understand why you are not including this part into the section, but I think it shouldn't be the main feature of Index manager as syncing data from discoverer does not provide any functionality to the user. 
What do you think?

Reply:
IMO your feedback is true and false.
Certainly, this does not give the benefit directly.
it's difficult because if this function does not work well, the user cannot get the effect of the `Vald Index Manager`.
So, both are correct and we have to choose one.
I'm fine whichever, but I appreciate it if you give the some suggestions.

Reply:
I think it also make sense. 
Considering this section is short, I think it should be fine without any changes on it.

Reply:
thanks :)
I won't update anymore about it.